src/profile/v100/entity/persona/persona_uniffi_fn.rs

```diff
@@ -25,6 +25,11 @@ pub fn new_persona_sample_mainnet_ripley() -> Persona {
     Persona::sample_mainnet_ripley()
 }
 
+#[uniffi::export]
+pub fn new_persona_sample_mainnet_turing() -> Persona {
```

Comment:
new sample values with `EntityFlags` with `DeletedByUser`, ensured other sample values do not have this flag, so that AuthorizedPersonaSimple references non hidden accounts and personas...